### DISCUSSION TITLE
Is it possible to render indicator near full width button label?

### DISCUSSION THREAD

**Discussion by BOHDANff** [mantinedev/mantine#5570] [Q&A] [answered]
Hello, I use full width button and want to have some indicator near button label. If I wrap such button in `Indicator` component indicator  will appear near the side of the button on the big distance from label. I try to wrap the button text by `Indicator` component but in such case Indicator displayed not near the label. So is there any way how to render indicator near full width button label?

Code example: [![Edit relaxed-diffie-jmncdy](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/p/sandbox/relaxed-diffie-jmncdy?file=%2Fsrc%2FApp.tsx%3A18%2C20)

**Comment by rtivital** [comment, marked as answer]
https://codesandbox.io/p/sandbox/funny-tu-qzsrp5?file=%2Fsrc%2FApp.tsx%3A14%2C67

**Reply by BOHDANff** [reply]
Thank you for help!
